core/scheduler/scheduler.go

```diff
@@ -493,15 +493,17 @@ func resolveActiveValidators(ctx context.Context, eth2Cl eth2wrap.Client,
 
 	var resp []validator
 	for index, val := range vals {
-		if !val.Status.IsActive() {
-			continue
-		}
-
 		pubkey, err := core.PubKeyFromBytes(val.Validator.PublicKey[:])
 		if err != nil {
 			return nil, err
 		}
 
+		instrumentValidator(pubkey, val.Validator.EffectiveBalance, val.Balance)
```

Comment:
sure, let's only instrument active validators. For inactive validators too, these values are defined but since the validator is inactive these values can be misleading.

Reply:
+1